src/TypeScript.UnitTests/Analyzer/EslintBridgeAnalyzerTests.cs

```diff
@@ -152,6 +152,75 @@ public async Task Analyze_EslintBridgeClientNotInitializedResponse_CallsInitLint
             client.Verify(x => x.InitLinter(It.IsAny<IEnumerable<Rule>>(), It.IsAny<CancellationToken>()), Times.Exactly(2));
         }
 
+        [TestMethod]
+        public async Task Analyze_EslintBridgeClientNotInitializedResponse_FollowedByParsingError_ParsingErrorIsLogged()
+        {
+            var logger = new TestLogger();
+            var linterNotInitializedResponse = CreateLinterNotInitializedResponse();
+            var parsingErrorResponse = new AnalysisResponse
+            {
+                ParsingError = new ParsingError
+                {
+                    Code = ParsingErrorCode.MISSING_TYPESCRIPT,
+                    Line = 5,
+                    Message = "some message"
+                }
+            };
+
+            var client = new Mock<IEslintBridgeClient>();
+
+            client
+                .SetupSequence(x => x.Analyze("some path", "some config", CancellationToken.None))
+                .ReturnsAsync(linterNotInitializedResponse)
+                .ReturnsAsync(parsingErrorResponse);
+
+            var testSubject = CreateTestSubject(client.Object, logger: logger);
+            await testSubject.Analyze("some path", "some config", CancellationToken.None);
+
+            logger.AssertPartialOutputStringExists(Resources.ERR_ParsingError_MissingTypescript);
+        }
+
+        [TestMethod]
+        public async Task Analyze_EslintBridgeClientNotInitializedResponse_FollowedByValidResponse_ConvertedIssuesReturned()
+        {
+            var linterNotInitializedResponse = CreateLinterNotInitializedResponse();
+            var validResponse = new AnalysisResponse {Issues = new List<Issue> {new Issue()}};
+
+            var client = new Mock<IEslintBridgeClient>();
+
+            client
+                .SetupSequence(x => x.Analyze("some path", "some config", CancellationToken.None))
+                .ReturnsAsync(linterNotInitializedResponse)
+                .ReturnsAsync(validResponse);
+
+            var convertedIssues = new[] { Mock.Of<IAnalysisIssue>() };
+            var issueConverter = new Mock<IEslintBridgeIssueConverter>();
+            SetupConvertedIssue(issueConverter, "some path", validResponse.Issues.First(), convertedIssues[0]);
+
+            var testSubject = CreateTestSubject(client.Object, issueConverter: issueConverter.Object);
+            var result = await testSubject.Analyze("some path", "some config", CancellationToken.None);
+
+            result.Should().BeEquivalentTo(convertedIssues);
```

Comment:
It wouldn't hurt to add some extra checks here e.g. that `analyze` is called twice, and that the "not initialized" message is not logged.